src/Cardano/Codec/Cbor.hs

```diff
@@ -41,10 +43,6 @@ module Cardano.Codec.Cbor
 
 import Prelude
 
-import Cardano.Address
-    ( Address (..), ProtocolMagic (..) )
-import Cardano.Address.Derivation
-    ( Depth (..), DerivationType (..), Index (..) )
```

Comment:
very good decision to make Cbor agnostic of our types and use Word32, ByteString